jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/surfaces/AppBarAssert.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.jdi.light.material.asserts.surfaces;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.material.elements.surfaces.AppBar;
+import org.hamcrest.Matchers;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public class AppBarAssert extends UIAssert<AppBarAssert, AppBar> {
+
```

Comment:
For App bar Assert I'd like to add following method:
   - Check that title text correct.
Please delete all unused methods